dotnet/SimpleSauce.Test/SauceOptionsTests.cs

```diff
@@ -107,12 +109,12 @@ public void AcceptsAllSauceLabsValues()
             SauceOptions.AvoidProxy.Should().BeTrue();
             SauceOptions.BuildName.Should().Be("Sample Build Name");
             SauceOptions.CapturePerformance.Should().BeTrue();
-            SauceOptions.ChromedriverVersion.Should().Be("71");
+            //SauceOptions.ChromedriverVersion.Should().Be("71");
             SauceOptions.Timeout.CommandTimeout.Should().Be(2);
             SauceOptions.CustomData.Should().BeEquivalentTo(customData);
             SauceOptions.ExtendedDebugging.Should().BeTrue();
             SauceOptions.Timeout.IdleTimeout.Should().Be(3);
-            SauceOptions.IeDriverVersion.Should().Be("3.141.0");
+            //SauceOptions.IeDriverVersion.Should().Be("3.141.0");
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/manual/SauceLabs/sauce_bindings/pullRequest?prid=6342070)